src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -285,28 +281,24 @@ public async Task<IActionResult> GetDisputeAsync(string guidHash, CancellationTo
     {
         try
         {
-            var token = HttpContext.Request.Headers.Authorization.FirstOrDefault();
-            if (String.IsNullOrEmpty(token))
-            {
-                throw new UnauthorizedAccessException("Invalid access_token");
-            }
-
             if (!_hashids.TryDecodeGuid(guidHash, out Guid noticeOfDisputeGuid))
             {
                 // TODO: add instrumentation to monitor invalid values being posted
                 return BadRequest("Invalid guidHash");
             }
 
+            var userInfoResponse = await _mediator.Send(GetCurrentUserInfoRequest.Default, cancellationToken);
+            UserInfo? user = userInfoResponse.UserInfo;

```

Comment:
treat it as access token, move to the top and check if user exist. if null, throw error